src/nplinker/loader.py

```diff
@@ -159,7 +159,24 @@ def _load_genomics(self):
         bigscape_cluster_file = (
             globals.BIGSCAPE_DEFAULT_PATH / f"mix_clustering_c{config.bigscape.cutoff}.tsv"
         )
-        raw_gcfs = BigscapeGCFLoader(bigscape_cluster_file).get_gcfs()
+        bigscape_db_file = (
+            globals.BIGSCAPE_DEFAULT_PATH / f"data_sqlite.db"
+        )
+
+        # switch depending on found file. prefer V1 if both are found
```

Comment:
It's ok to use this logics at the moment. Later it should be adapted e.g. with a new config on bigscape version.